babel.config.js

```diff
@@ -8,9 +8,8 @@ const isBrowser = isCalypsoClient || 'true' === process.env.TARGET_BROWSER;
 const modules = isBrowser ? false : 'commonjs'; // Use commonjs for Node
 const codeSplit = require( './server/config' ).isEnabled( 'code-splitting' );
 
-const targets = isBrowser
-	? { browsers: [ 'last 2 versions', 'Safari >= 10', 'iOS >= 10', 'ie >= 11' ] }
-	: { node: 'current' };
+// Use target configuration in package.json for browser builds.
+const targets = isBrowser ? undefined : { node: 'current' };
```

Comment:
Since this PR has been open for a while and this line gets tossed around:

- https://github.com/Automattic/wp-calypso/pull/31464/files#diff-0e759b21864de1336af9c450cff5ad20R9
- https://github.com/Automattic/wp-calypso/pull/31536#issuecomment-474236981
- https://github.com/Automattic/wp-calypso/pull/31503/files#diff-f0bc8d1120b1704c18d6a492ba4ee38fR6

... it would be good to PR this change separately.

Reply:
Landed in https://github.com/Automattic/wp-calypso/pull/31561